src/content/guides/prepare-azure-account-for-guest-clusters/index.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Prepare an Azure account to run Giant Swarm guest clusters"
```

Comment:
I think we should use `subscription` where we use `account` now, in the entire article.